File: pocketr-java.api/src/main/java/com/decrux/pocketr/api/repositories/HouseholdRepository.java
package com.decrux.pocketr.api.repositories;

import com.decrux.pocketr.api.entities.db.household.Household;
import java.util.UUID;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface HouseholdRepository extends JpaRepository<Household, UUID> {
}

File: pocketr-api/src/main/kotlin/com/decrux/pocketr_api/repositories/HouseholdRepository.kt
package com.decrux.pocketr_api.repositories

import com.decrux.pocketr_api.entities.db.household.Household
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface HouseholdRepository : JpaRepository<Household, UUID>
